blog/2020-03-01-CombiningBPMNWithOptaplanner.adoc

```diff
@@ -0,0 +1,611 @@
+= Combining BPMN with Optaplanner
+:page-interpolate: true
+:awestruct-author: Christopher-Chianelli
+:awestruct-layout: blogPostBase
+:awestruct-tags: [useCase]
+:awestruct-date: 2020-03-01
+
+Many Business Processes modelled with BPMN diagrams. When the Business
+Process involves optimization at some step,
+https://www.optaplanner.org/[OptaPlanner] can perform that task for us. Now with
```

Comment:
"us" -> Use active tense. If that doesn't fit, use "you", by exception. Don't use "we" or "us", "I" or "me".

" When x involves y, z can perform that task for us." That sentence isn't convincing. It doesn't show me the benefit. It's explaining the benefit indirectly.